### PR TITLE
Fixed class name in FlattenException serializer configuration

### PR DESCRIPTION
**Pull request by chq81**
Hi @kunicmarko20 . I accidentally introduced a bug into the last PR, with the class name not matching the type of the exception class.

### CODE REVIEW

File: src/Resources/config/serializer/Symfony.Component.Debug.Exception.FlattenException.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <serializer>
-    <class name="Symfony\Component\Messenger\Stamp\RedeliveryStamp" exclusion-policy="ALL">
+    <class name="Symfony\Component\Debug\Exception\FlattenException" exclusion-policy="ALL">
```
Comment (kunicmarko20) [inline src/Resources/config/serializer/Symfony.Component.Debug.Exception.FlattenException.xml:3]: can you add debug as a dev dependency please

Comment (chq81) [reply src/Resources/config/serializer/Symfony.Component.Debug.Exception.FlattenException.xml:3]: > can you add debug as a dev dependency please

Symfony/Debug is already a dev dependency.

**Comment by kunicmarko20** [comment]
Thank you @chq81 !

**Comment by kunicmarko20** [comment]
https://github.com/kunicmarko20/jms-messenger-adapter/releases/tag/0.2.2

**Pull request merged**